public/js/scripts.js

```diff
@@ -389,7 +391,7 @@ function calculateMapSize() {
       -latStep;
 
     // TODO this shouldn’t be hardcoded, but it is. Sue me.
-
+    /*
     switch (cityId) {
       case "africa":
         globalScale *= 0.8;
```

Comment:
We could finally remove this hardcoded block at all :partying_face:

Reply:
Yes - though I thought removing this block could be something done later, once we're sure we're done with it.

Reply:
I'm fine with that!